src/Microsoft.OData.Edm/Schema/EdmModelBase.cs

```diff
@@ -175,23 +197,27 @@ public virtual IEnumerable<IEdmOperation> FindDeclaredBoundOperations(string qua
         {
             IEnumerable<IEdmOperation> enumerable = this.FindDeclaredOperations(qualifiedName);

```

Comment:
>FindDeclaredOperations [](start = 57, length = 22)

Rather than getting operations based on name, and filtering by binding type, now that we are caching the declared operations by binding type, we should call that method and filter the results by name.